### PR TITLE
Usecases

### PR DESCRIPTION
**Pull request by deepakachu-aiplanet**
added youtube study planner usecase

### CODE REVIEW

File: docs/use-cases/youtube_studyplan_agent.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+# Run the agent to generate a study plan
+admin.run(
+    query="""I want to study optimization techniques.""",
+    description="""Create a detailed study plan for studying optimization techniques with all major topics listed, 
+    along with a reference YouTube video for each topic."""
+)
+```
+
+## 📝 Sample Output
```
Comment (lucifertrj) [inline docs/use-cases/youtube_studyplan_agent.md:100]: rename to Output

File: docs/use-cases/youtube_studyplan_agent.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+## 🧑‍💼 Admin and Workers
+
+Set up the Admin agent, which manages the workflow and assigns tasks to worker agents. The admin ensures that the study plan is output in markdown format.
+
+```python
+# Define the admin
+admin = Admin(
+    llm=llm,
+    planner=planner,
+    max_steps=5,
```
Comment (lucifertrj) [inline docs/use-cases/youtube_studyplan_agent.md:60]: Admin have no max_steps=5, replace with max_iterations

File: docs/use-cases/youtube_studyplan_agent.md (hunk 1 above)
Comment (deepakachu-aiplanet) [reply docs/use-cases/youtube_studyplan_agent.md]: done

File: docs/use-cases/youtube_studyplan_agent.md (hunk 2 above)
Comment (deepakachu-aiplanet) [reply docs/use-cases/youtube_studyplan_agent.md]: done, but i think we have to change this on docs too:
image

**Pull request closed without merging**